app/src/main/java/org/jellyfin/androidtv/playback/PlaybackController.java

```diff
@@ -615,47 +608,18 @@ public void run() {
         mApplication.setLastPlayedItem(item);
         if (!isRestart) ReportingHelper.reportStart(item, mbPos);
         isRestart = false;
-
-        //test
-//        mHandler.postDelayed(new Runnable() {
-//            @Override
-//            public void run() {
-//                mVideoManager.fakeError();
-//            }
-//        }, 15000);
-
     }
 
     private int getDefaultAudioIndex(StreamInfo info) {
         return mPlaybackMethod != PlayMethod.Transcode && info.getMediaSource().getDefaultAudioStreamIndex() != null ? info.getMediaSource().getDefaultAudioStreamIndex() : -1;
     }
 
     public void startSpinner() {
-        if (mApplication.getCurrentActivity() != null) {
-            mApplication.getCurrentActivity().runOnUiThread(new Runnable() {
-                @Override
-                public void run() {
-                    if (mSpinner != null) mSpinner.setVisibility(View.VISIBLE);
-                    spinnerOff = false;
-                }
-            });
-
-        }
-
+        spinnerOff = false;
```

Comment:
Seems like this may not be needed now?

Reply:
It is actually used for something else and I did not want to break that code